<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addDemFrm</class>
 <widget class="QWidget" name="addDemFrm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加高程</string>
  </property>
  <widget class="QPushButton" name="cancleBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>11</y>
     <width>661</width>
     <height>141</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>单幅DEM</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>5</y>
       <width>631</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>GDAL驱动</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>32</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>DEM名称：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>74</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>选择路径：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="demPathGdalBtn">
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>72</y>
        <width>461</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="demNameGdalBtn">
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>30</y>
        <width>461</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="selectPathGdalBtn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>70</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>选择..</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>DEM切片</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>5</y>
       <width>631</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>TMS驱动</string>
     </property>
     <widget class="QPushButton" name="selectPathTmsBtn">
      <property name="geometry">
       <rect>
        <x>563</x>
        <y>68</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>选择..</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="demNameTmsBtn">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>24</y>
        <width>481</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="demPathTmsBtn">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>68</y>
        <width>481</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>26</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>配置文件：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>70</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>选择路径：</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
